tests/tr1/tests/memory1/test.cpp

```diff
@@ -13,6 +13,9 @@
 #include <sstream>
 #include <string>
 
+// DevDiv 1281038: False C6287 for C++20 synthesized operators

```

Comment:
What is the correct method to reference DevDiv bugs?

Reply:
Will update to VSO-XXXXXX along with any other feedback.

Reply:
We also conventionally say `// TRANSITION, VSO-num` - the `TRANSITION` comments are highly searchable, and distinguish bug citations for workarounds that should be removed eventually, from bug citations for regression tests.